README.zh-CN.md

```
@@ -85,16 +85,18 @@ Pro Series 组件经过精心设计，以增强企业级应用程序的健壮架
 
 ## 🖥 浏览器兼容性
 
-> \[!NOTE]
->
-> - 现代浏览器和 Internet Explorer 11（使用[兼容性填充](https://stackoverflow.com/questions/57020976/polyfills-in-2019-for-ie11)）
-> - [Electron](https://www.electronjs.org/)
+支持范围：https://browsersl.ist/#q=defaults
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)